src/utils/proxyquire.js

```diff
@@ -26,33 +16,68 @@ const getJestMockStatement = ({ j, mockName, mockBody }) =>
         ])
     );
 
+function getIdentifierValue(ast, name) {
+    const varDec = ast.findVariableDeclarators(name).get(0);
+
+    if (varDec && 'value' in varDec.node.init) {
+        return varDec.node.init.value;
+    }
+
+    return null;
+}
+
 export default function proxyquireTransformer(fileInfo, j, ast) {
     const importVariableName = removeRequireAndImport(j, ast, 'proxyquire');
     if (importVariableName) {
         const mocks = new Set();
 
         ast
-            .find(j.Identifier, {
-                name: importVariableName,
-            })
-            .forEach(p => {
-                const outerCallExpression = findFirstParentCallExpression(p);
-                if (!outerCallExpression) {
-                    return;
+            .find(j.CallExpression, match => {
+                if (
+                    match.callee.type === 'CallExpression' &&
+                    match.callee.callee.type === 'MemberExpression'
+                ) {
+                    return match.callee.callee.object.name === importVariableName;
+                } else if (
+                    match.callee.type === 'MemberExpression' &&
+                    match.callee.object.type === 'CallExpression' &&
+                    match.callee.object.callee.type === 'MemberExpression'
+                ) {
+                    return match.callee.object.callee.object.name === importVariableName;
+                } else if (
+                    match.callee.type === 'MemberExpression' &&
+                    match.callee.object.name === importVariableName
+                ) {
+                    const parentFind = ast.find(j.CallExpression, { callee: match });
+                    const parent = parentFind.size() && parentFind.get();
+
+                    return !parent || parent.node.type !== 'CallExpression';
                 }
 
+                return match.callee.name === importVariableName;
+            })
+            .forEach(outerCallExpression => {
                 const args = outerCallExpression.node.arguments;
                 if (args.length === 0) {
                     // proxyquire is called with no arguments
                     j(outerCallExpression).remove();
                     return;
                 }
 
-                const requireFile = args[0].value;
+                const pathArg = args[0];
+                const requireFile =
```

Comment:
In my personal version I `return;` here when `requireFile` is falsey - wasn't sure whether that fits the mold here.